lib/diagnostic.js

```diff
@@ -38,18 +52,34 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = {
```

Comment:
The values for each key needs to be wrapped in quotation marks since they are supposed to be strings.  Right now, the interpreter will think that they're names of things that don't exist and will throw an error.